dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -70,7 +70,7 @@ def adjust_facilitators
 
     new_facilitator_ids.each do |facilitator_id|
       facilitator = User.find_by(id: facilitator_id)
-      next unless facilitator.facilitator?
+      next unless facilitator && facilitator.facilitator?
```

Comment:
no, but these ids are supplied by the caller so they could feasibly be any user id. I think potential_facilitator_id is a bit much ;). I'll add a comment.